test/test_torch.py

```diff
@@ -18633,6 +18638,8 @@ def inner(self, device, dtype):
     ('exp', '', _small_3d, lambda t, d: [], 1e-2, 1e-2, 1e-5, _float_types),
     ('exp', 'samll', lambda t, d: _small_3d(t, d).clamp(-1, 1),
         lambda t, d: [], 1e-2, 1e-2, 1e-5, _float_types, [torch.bfloat16]),
+    ('exp2', 'samll', lambda t, d: _small_3d(t, d).clamp(-1, 1),
```

Comment:
"samll" -> "small", would you correct this in the preceeding exp test case, too?

Also, you could add another test cases analogous to the first exp test case:

```
('exp2', '', _small_3d, lambda t, d: [], 1e-2, 1e-2, 1e-5, _float_types),
```